theano/sandbox/cuda/tests/test_cula.py

```diff
@@ -7,18 +7,18 @@
 # Skip tests if cuda_ndarray is not available.
 from nose.plugins.skip import SkipTest
 import theano.sandbox.cuda as cuda_ndarray
-if not cuda_ndarray.cuda_available:
-    raise SkipTest('Optional package cuda not available')
 from theano.misc.pycuda_init import pycuda_available
 from theano.sandbox.cuda.cula import cula_available
 
+from theano.sandbox.cuda import cula
```

Comment:
That's a bummer because not reordering makes flake8 fail.  I'll try to have an alternate solution.